var/spack/repos/builtin/packages/dnstracer/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Dnstracer(AutotoolsPackage):
```

Comment:
This package doesn't actually use Autotools. I would switch to the `Package` base class and add `build` and `install` methods.